crates/uv-build-frontend/src/error.rs

```diff
@@ -68,19 +69,47 @@ pub enum Error {
     #[error("Editable installs with setup.py legacy builds are unsupported, please specify a build backend in pyproject.toml")]
     EditableSetupPy,
     #[error("Failed to resolve requirements from {0}")]
-    RequirementsResolve(&'static str, #[source] anyhow::Error),
+    RequirementsResolve(&'static str, #[source] AnyErrorBuild),
     #[error("Failed to install requirements from {0}")]
-    RequirementsInstall(&'static str, #[source] anyhow::Error),
+    RequirementsInstall(&'static str, #[source] AnyErrorBuild),
     #[error("Failed to create temporary virtualenv")]
     Virtualenv(#[from] uv_virtualenv::Error),
+    // Build backend errors
     #[error("Failed to run `{0}`")]
     CommandFailed(PathBuf, #[source] io::Error),
-    #[error(transparent)]
+    #[error("The build backend returned an error. This likely means a problem with the package or your environment.")]
```

Comment:
I'd rephrase a little (applies elsewhere)

```suggestion
    #[error("The build backend returned an error; this usually indicates a problem with the package or your environment.")]
```